src/abstract.js

```diff
@@ -793,16 +793,93 @@ Sk.abstr.sattr = function (obj, nameJS, data, canSuspend) {
 };
 goog.exportSymbol("Sk.abstr.sattr", Sk.abstr.sattr);
 
-Sk.abstr.iter = function (obj) {
+
+Sk.abstr.iternext = function (it, canSuspend) {
+    return it.tp$iternext(canSuspend);
+};
+goog.exportSymbol("Sk.abstr.iternext", Sk.abstr.iternext);
+
+
+/**
+ * @constructor
+ */
+function seqIter(obj) {
```

Comment:
couldn't we just move it inside `Sk.abstr.iter`? I'll check when I review this tonight. We haven't establishes such a convention, are there many of these "classes"? I don't remember any. :smile:.